### DISCUSSION TITLE
Harbor CLI

### DISCUSSION THREAD

**Discussion by syslab007** [goharbor/harbor#19777] [General]
Harbor can provide CLI functionality.  To use commands through access harbor and all features like search project delete repository etc.

**Comment by zyyw** [comment]
You can send HTTP requests to interact with your Harbor instance.

**Reply by syslab007** [reply]
Thanks for the replay. How I interact with API I have shown the API but we require that the repository's latest tag with untag store till 5 and the custom tag be stored other all repository images will be deleted.

**Reply by zyyw** [reply]
Also please reference this:
* https://github.com/hinyinlam/cli-for-harbor
